spec/system/candidate_entering_contact_details_spec.rb

```diff
@@ -11,16 +11,16 @@
       click_on t('application_form.save_and_continue')
     end
 
-    it 'sees a summary of those details' do
-      visit '/check_your_answers'
+    describe 'visiting the check_your_answers page' do
+      before do
+        visit '/check_your_answers'
+      end
 
-      expect(page).to have_content('Phone number 1234567890')
-    end
+      it 'sees a summary of those details' do
```

Comment:
I think this stuff is already in `master` and ought to rebase out cleanly

Reply:
It is, I think it that's because I changed the indentation, and it looks like I added it.